deploy.sh

```diff
@@ -29,10 +24,24 @@ main() {
 		elif [[ $1 = "-n" || $1 = "--no-hash" ]]; then
 			append_hash=false
 			shift
+		elif [[ $1 = "-c" || $1 = "--config-file" ]]; then
```

Comment:
Thought about it a little and... should we check that `$2` is defined (as is done with `-m`)? Or is an error like `./deploy.sh: line 28: <string>: No such file or directory` good enough?